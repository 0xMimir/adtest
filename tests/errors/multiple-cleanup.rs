fn main(){

}

#[adtest::adtest(cleanup = cleanup, cleanup = cleanup)]
fn invalid_test(){

}

fn cleanup(){

}