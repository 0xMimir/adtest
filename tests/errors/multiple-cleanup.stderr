error: Cleanup already defined
 --> tests/errors/multiple-cleanup.rs:5:1
  |
5 | #[adtest::adtest(cleanup = cleanup, cleanup = cleanup)]
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the attribute macro `adtest::adtest` (in Nightly builds, run with -Z macro-backtrace for more info)
